tests/ops/service/connectors/test_queryconfig.py

```diff
@@ -141,7 +141,7 @@ def test_generated_sql_query(self):
                     }
                 )
             )
-            == "SELECT id,name,ccn,customer_id,billing_address_id FROM payment_card WHERE id = :id OR customer_id = :customer_id"
+            == "SELECT billing_address_id,ccn,customer_id,id,name FROM payment_card WHERE id = :id OR customer_id = :customer_id"
```

Comment:
The order changed after the fixture refactor but it still seems deterministic, I think this is _ok_.